.github/workflows/rust.yml

```diff
@@ -136,6 +136,26 @@ jobs:
             --
             -D warnings
 
+  rustfmt:
+    name: Rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
`format_code_in_doc_comments` is the `nightly` channel only.

ref:
[format_code_in_doc_comments](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=format#format_code_in_doc_comments)